### PR TITLE
fix(docs): update ADR docs for structure, standards, clarity, completeness

### PR DESCRIPTION
**Pull request by barriebyron**
This PR organizes and improves our ADR documentation:
- add TOC to the README as specified in the PROCESS doc
- apply style and standards
- I want to remove duplicate document since we have 2 DID ADR docs, is the correct DID doc the 002 or the 003?
- fix typos and so on

A little editorial and developmental editing love for our most important architectural decision docs.

### CODE REVIEW

**Comment by barriebyron** [comment]
looking for input on which ADR for DID is the canonical true ADR doc, see my [question on Discord](https://discord.com/channels/988370465179451432/988375014111870987/1016727292711546972)  @noandrea and who else can weigh in? @SenorMonito perhaps? (I would love a PRP list... who is our primary responsible party for the product components?)

File: docs/Explanation/ADR/adr-002-did.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# ADR 002: DID
+# ADR 002: Decentralized Identifiers (DIDs)
```
Comment (spoo-bar) [inline docs/Explanation/ADR/adr-002-did.md:5]:
```suggestion
---
title: ADR 002:  Decentralized Identifiers (DIDs)
---

# ADR 002: Decentralized Identifiers (DIDs)
```

**Review by alessio — APPROVED** [review]
Love this :)

**Comment by barriebyron** [comment]
thank you @spoo-bar  for your feedback! Who else can weigh in on these changes? 
I'd like to see these quality-focused updates go live.
@noandrea can you peek?

**Review by noandrea — APPROVED** [review]
big :+1: from me. 
For the adr-8 there is going to be some more adjustments to make due to https://github.com/elesto-dao/elesto/pull/239

**Pull request closed without merging**